article/app/services/RenderingTierPicker.scala

```diff
@@ -30,17 +30,25 @@ object RenderingTierPicker {
   }
 
   private def hasOnlySupportedElements(page: PageWithStoryPackage): Boolean = {
```

Comment:
how about
```
private def hasOnlySupportedElements(page: PageWithStoryPackage): Boolean = {
    def unsupportedElement(blockElement: BlockElement) = blockElement match {
      case _: TextBlockElement => true
      case _: ImageBlockElement => true
      case _ => false
    }
    !page.article.blocks.exists(_.body.exists(_.elements.exists(unsupportedElement)))
  }
```
?

Reply:
Yeah that looks better, will change and test